src/players/VAST.js

```diff
@@ -0,0 +1,211 @@
+import React, { Component } from 'react'
+import vast from 'vast-client'
+import { callPlayer } from '../utils'
+import createSinglePlayer from '../singlePlayer'
+import { FilePlayer } from './FilePlayer'
+
+const MATCH_URL = /^(https?:\/\/)?((bs\.serving-sys\.com)|(pubads\.g\.doubleclick\.net))/
```

Comment:
change this to be insensitive:
`const MATCH_URL = /^(https?:\/\/)?((bs\.serving-sys\.com)|(pubads\.g\.doubleclick\.net))/i`